cmd/job-trigger-controller-manager/main.go

```diff
@@ -0,0 +1,113 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/bombsimon/logrusr"
```

Comment:
Why do we need this? It does not look like the other our controllers use this, what makes this one special?

Reply:
This is the magic moment, where I accidentally add an extra `r` in `logrus` and vscode finds a new library. This is hilarious

Reply:
Ah no.. my bad.. We are already using this library. Check the other controllers. See for example: https://github.com/openshift/ci-tools/blob/master/cmd/ci-operator/main.go#L26 or https://github.com/openshift/ci-tools/blob/master/cmd/dptp-controller-manager/main.go#L13